apps/editing-toolkit/editing-toolkit-plugin/error-reporting/index.php

```diff
@@ -117,6 +117,7 @@ function enqueue_script() {
 		'A8C_ETK_ErrorReporting_Config',
 		array(
 			'shouldActivateSentry' => should_activate_sentry( get_current_user_id(), get_current_blog_id() ) ? 'true' : 'false',
+			'releaseName'          => 'WPCOM_' . WPCOM_DEPLOYED_GIT_HASH,
```

Comment:
If you check 2a9fb7a3f114573891153d47e6bcf7793c4e57ea, you'll see I had originally wrapped this expression in a ternary. The rationale was that ETK is pushed to the org repo and is not entirely WPCOM-specific. However, it didn't make much sense: this code only really runs in WPCOM and is WPCOM-specific. When we get to the point of offering this a feature for org sites, then we, yes, we need to change this logic and abstract this interface, but for now, the constant WILL be defined in WPCOM.

Reply:
It also might be better to define another constant `wpcom-options.php` (see D84565-code) immediately after the `WPCOM_DEPLOYED_GIT_HASH`. The constant can be called `WP_ERROR_REPORTING_RELEASE_NAME` or something along the lines, and will be defined as:

```php
'WPCOM_' . WPCOM_DEPLOYED_GIT_HASH
```

Like above. We can then just pass it over in the `releaseName` property above. Two main benefits of doing this:
* The interface becomes less semantically-tied to WPCOM, so if we want to generalize this feature in the future, it will be easier
* We don't need to concatenate the `WPCOM_` prefix in every place we need this name. Less error-prone! 

I'll look into doing this shortly.

Reply:
Nah, I've been looking into this, and due to the way the post-deploy hook work, it'll be simpler just to concatenate everywhere we need instead of having a single constant. The idea was to also use this constant in the post-deploy script that creates the Sentry release, but I don't know at which point in the deployment step the file is created. It's possible I'd be referencing the previous release hash. I might explore this in the future, but for now, it's simpler just to concatenate the prefix everywhere.